### DISCUSSION TITLE
What does slip parameters signify in FEPX configuration files?

### DISCUSSION THREAD

**Discussion by neelammishra** [thelfer/tfel#174] [Q&A] [answered]
m 0.010d0 (fixed state strain rate sensitivity)
    gammadot_0 1.0d0 (fixed state strain rate  scaling coefficient)
    h_0 190.0d0 (strength hardening rate coefficient)
    g_0 390.0d0 468.0d0 663.0d0 (initial slip system strength)
    g_s0 530.0d0   ?????????
    m_prime 1.10d0 (saturation strength rate scaling component)
    gammadot_s0 1.0d0  (saturation strength strain rate scaling coefficient)
    n 1.0d0 ?????????

what does d0 mean here??    g_s0 ??  n??????

**Comment by thelfer** [comment, marked as answer]
@neelammishra  Could you provide a bit of context, I have absolutly no clue of what you are taking about :)

**Reply by neelammishra** [reply]
sorry. I posted in the wrong forum
